### ISSUE TITLE
Range value of arousal, valence, dominance

### ISSUE THREAD

**Issue by l4zyf9x** [audeering/w2v2-how-to#22]
I wonder what is the range value of arousal, valence, and dominance. As far as I know, model output is a logit vector size of 3 representing that feature. I see that you use MSP-Conversation Corpus for fine-tuning. But when I looked at The MSP-Conversation Corpus paper [paperlink](https://ecs.utdallas.edu/research/researchlabs/msp-lab/publications/Martinez-Lucas_2020.pdf), they mentioned that 
"Notice that the values of the traces are in the range between -100 and 100. The figure shows that extreme values are uncommon. Most of the annotations are concentrated between -40 to 40 for valence, -20 to 50 for arousal, and -20 to 40 for dominance"

Do you guys normalize that feature, or do something related?

**Comment by hagenw** [comment]
Yes, databases tend to use different scales for arousal/valence/dominance like 0..5.
We normalize all scales to 0..1 for training. During inference most of the values returned by the model are in this range, but it can happen that you also get some values outside of that range.

**Comment by bahtuchi** [comment]
I just created a 10-second, completely silent .wav file to test the model. If i understood correctly, the values should all be 0.5, i.e. completely neutral, right? 
I used the following setting: 
`interface = audinterface.Feature(
    model.labels(‘logits’),
    process_func=model,
    process_func_args={
        ‘outputs’: ‘logits’,
    },
    sampling_rate=sampling_rate,
    process_func_applies_sliding_window=False,
    win_dur=1,
    hop_dur=0.5,
    resample=True,
    verbose=True,
)
`
I got 10 values, i.e. arousal, dominance and valence and they were always `(0.55; 0.61; 0.64)`. How can this be explained?
I mean it is good that there isnt any variation but I am a bit confused why they are not all 0.5.

**Comment by hagenw** [comment]
The model was not trained on non-speech input (like silence) and might hence not be able to abstract to other inputs like silence or sound of objects. It is assumed that you always use a voice activity detection (VAD) and pass on only speech to the model.
